League Sandbox Auto Setup/leagueSandboxAutoSetupForm.cs

```diff
@@ -134,19 +134,19 @@ private void startCloningRepositories()
                         }));
                     }
                 };
-                options.BranchName = "Indev";
                 options.RecurseSubmodules = false;                
 
 //                git submodule init
-//git submodule update
+//                git submodule update
 
                 if (!Directory.Exists(cloningPath))
                 {
                     Directory.CreateDirectory(cloningPath);
+                    options.BranchName = "Indev";  // Branch for GameServer
                     Repository.Clone("https://github.com/LeagueSandbox/GameServer", cloningPath, options);
-                    options.BranchName = "master";
+                    options.BranchName = "master"; // Branch for LeaguePackets
                     Repository.Clone("https://github.com/LeagueSandbox/LeaguePackets", Path.Combine(cloningPath, "LeaguePackets"), options);
-                    options.BranchName = "indev";
+                    options.BranchName = "Indev"; // Branch for LeagueSandbox-Default
```

Comment:
I'm not sure why you have changed `indev->Indev`. Git branch names in general I think are case sensitive, but seems that this is not the case on Windows. I suggest to change `Indev->indev`.

Reply:
Done, could we switch whole repos to use master / **main** branch, as the most stables ones for future development?

Reply:
There is no need to switch.